src/test/java/co/rsk/federate/signing/hsm/message/ReleaseCreationInformationGetterTest.java

```diff
@@ -69,7 +69,15 @@ public void createGetTxInfoToSign_returnOK() throws HSMReleaseCreationInformatio
                 receiptStore,
                 blockStore
         );
+        // HSM V2
+        createGetTxInfoToSign_returnOK_v2(information, rskTxHash, btcTransaction, block, transactionReceipt);
 
+        // HSM V3
+        createGetTxInfoToSign_returnOK_v3(information, rskTxHash, btcTransaction, block, transactionReceipt);
+    }
+
+    public void createGetTxInfoToSign_returnOK_v2(ReleaseCreationInformationGetter information, Keccak256 rskTxHash,
```

Comment:
What if you add the version as an argument? That way you only need one method

Reply:
Thanks Nath, Resolved